forui/lib/src/widgets/button/button_content.dart

```diff
@@ -31,14 +20,16 @@ final class FButtonContent extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final style = this.style.content;
-
     return Padding(
       padding: style.padding,
       child: Row(
         mainAxisAlignment: MainAxisAlignment.center,
         children: [
           if (icon != null) ...[
-            icon!(height: 20, colorFilter: enabled ? style.enabledIcon : style.disabledIcon),
+            icon!(
+              height: 20,
+              colorFilter: ColorFilter.mode(enabled ? style.enabledIcon : style.disabledIcon, BlendMode.srcIn),
```

Comment:
Why not just store the ColorFilter object?

Reply:
I thought about it, and it felt more convenient to expose a color rather than a `ColorFilter`. Most users will only be interested in changing the color and not the `BlendMode`.